src/parsers/manifest/dash/js-parser/parse_from_document.ts

```diff
@@ -92,6 +96,10 @@ export default function parseFromDocument(
                       receivedTime,
                       sendingTime,
                       url } = loadedXlinks[i];
+              if (typeof xlinkData !== "string") {
+                const newRet = initialRes.value.continue([]);
```

Comment:
I think we should still fail (preferably with the same error than before) if we're not able to fetch xlinks